core/src/utils/directive.ts

```diff
@@ -1,7 +1,11 @@
 import type {ReadableSignal} from '@amadeus-it-group/tansu';
 import {asReadable, batch, readable, writable} from '@amadeus-it-group/tansu';
+import {BROWSER} from 'esm-env';
```

Comment:
If you use `esm-env`, you should add it to the peer dependencies (or dependencies) of the core and check it is part of the external rollup options in `vite.config.ts`. Otherwise, vite will bundle it with the core with `BROWSER` hard-coded to `true` and it no longer does what you expect.